src/client/styles/components/_trip.scss

```diff
@@ -27,8 +27,8 @@
     margin-right: 5%;
 
     img {
-      width: 22vw;
-      height: 22vw;
+      width: 100%;
+      height: 100%;
```

Comment:
will this cause the image ratio to not be 1:1?

Reply:
It appeared to be 1:1 on local testing